standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -391,6 +395,15 @@ public void setConf(Configuration conf) {
     } else {
       LOG.debug("Initialized ObjectStore");
     }
+
+    if (tablelocks == null) {
+      synchronized (ObjectStore.class) {
+        if (tablelocks == null) {
+          int numTableLocks = MetastoreConf.getIntVar(conf, ConfVars.METASTORE_NUM_STRIPED_TABLE_LOCKS);
+          tablelocks = Striped.lock(numTableLocks);
```

Comment:
Is this different from ReentrantLock? (PaddingLock extends ReentrantLock and adds padding for cache line). But not heavily benchmarked. E.g https://github.com/google/guava/issues/3195

Changing to ReenetrantLock may be easier to read?

Reply:
sure, @rbalamohan thanks for taking a look at this!

Reply:
@rbalamohan, are you ok with the change?